client/src/App.js

```diff
@@ -1,13 +1,14 @@
 import { useAuth0 } from "@auth0/auth0-react";
 import { Route, Routes } from "react-router-dom";
 import { Chakra } from "./components/Chakra";
-
 import Layout from "./components/layout";
 import Loading from "./components/Loading";
 import LoginPage from "./pages/LoginPage";
-import Home from "./pages/home";
-import Users from "./pages/users/";
+import About from "./pages/About";
+import Home from "./pages/Home";
+import Users from "./pages/users/Users";
```

Comment:
This React component doesn't exist on the remote repository, potentially a commit issue?